CodeBenders/Assets/Scripts/Block.cs

```diff
@@ -11,11 +11,17 @@ public class Block : MonoBehaviour
     /// </summary>
     public string woodenBlockTag;
 
+    public bool inside_grid;
```

Comment:
I might change the variable name to conform with the CamelCase convention of C#, and add a comment. But this is a minor issue which can be fixed with automatic refactors. Otherwise looks good to me.